src/jvmMain/kotlin/io/beatmaps/api/alerts.kt

```diff
@@ -99,7 +101,14 @@ fun Route.alertsRoute() {
                 } > 0
             }
 
-            call.respond(if (result) HttpStatusCode.OK else HttpStatusCode.BadRequest)
+            if (result) {
+                call.sessions.set(
+                    user.copy(alerts = user.alerts?.let { if (req.read) it - 1 else it + 1 })
```

Comment:
I would probably just query the new value in case an alert has been added since the last time we checked